packages/forklift-console-plugin/src/components/page/StandardPage.tsx

```diff
@@ -350,6 +350,27 @@ export function StandardPage<T>({
     .filter((field) => field.filter?.primary)
     .map(toFieldFilter(sortedData));
 
+  const onSetPage: (
+    _event: React.MouseEvent | React.KeyboardEvent | MouseEvent,
```

Comment:
```suggestion
    event: React.MouseEvent | React.KeyboardEvent | MouseEvent,
```

no need for `_` in the type definition

Reply:
Correct, but this is how it was declared in PF for the function so I wanted to keep the same signature as much as possible.  
https://github.com/patternfly/patternfly-react/blob/db0cdde6e3e6ae61348129b73bb43c97a98b4617/packages/react-core/src/components/Pagination/Pagination.tsx#L82

Anyway, I can see now that it was fixed for PF react 5, so will fix it in our code as well.